python/ray/train/v2/api/callback.py

```diff
@@ -0,0 +1,53 @@
+from typing import Any, Dict, List, Optional
+
+from ray.train import Checkpoint
+from ray.train.v2._internal.execution.context import TrainRunContext
+from ray.util.annotations import DeveloperAPI
+
+
+@DeveloperAPI
+class RayTrainCallback:
+    """Base Ray Train callback interface."""
+
+    pass
+
+
+@DeveloperAPI
+class UserCallback(RayTrainCallback):
```

Comment:
Naming?

Reply:
`UserCallback` sounds good & straightforward to me.